### PR TITLE
Add release notes for 0.12.1

### PR DESCRIPTION
**Pull request by Jaanki**
Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

**Comment by dfarrell07** [comment]
I was updating the GDoc as you sent this PR :stuck_out_tongue_winking_eye:. Can you copy the latest from there?

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+### Bug fixes
```
Comment (maayanf24) [inline src/content/community/releases/_index.en.md]: we can follow the same format we already used in previous point releases : https://submariner.io/community/releases/#v0111

Comment (maayanf24) [inline src/content/community/releases/_index.en.md]: we can follow the same format we already used in previous point releases : https://submariner.io/community/releases/#v0111

**Comment by nyechiel** [comment]
@Jaanki we are already working on releasing 0.12.2. Can you please get this one reviewed and merged as soon as possible? thanks!

**Review by maayanf24 — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+This is a bugfix release:
+
+* The default image type for a dedicated gateway node is changed to a smaller one for OpenStack Cloud prepare.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]: Can we specify what is the actual type? "smaller one" is not very descriptive IMO.

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,17 @@ title = "Releases"
...
+* `subctl gather` will now use `libreswan` as  a default cable driver if none is specified in `SubmarinerSpec` during installation.
+* Sometimes when Submariner is used to connect onPrem clusters with Public clusters, MTU issues are seen. This was particularly noticed
+when the underlying platform uses `nftables` on the host nodes. This release fixes the MTU issues by explicitly clamping the TCP MSS to a
+fixed value derived from the default interface MTU subtracted with the cable-driver overhead.
+* As part of `subctl uninstall` operation, we now remove the `submariner.io/globalIp` annotation that is added on the Gateway node.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]:
```suggestion
* As part of `subctl uninstall` operation, we now remove the `submariner.io/globalIp` annotation that is added on the gateway node.
```

Comment (nyechiel) [reply src/content/community/releases/_index.en.md]: I am not sure if we should capitalize "gateway" or not (here for e.g we do: https://submariner.io/getting-started/architecture/gateway-engine/), but at least we should make it consistent (see L11).

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+This is a bugfix release:
+
+* The default image type for a dedicated gateway node is changed to a smaller one for OpenStack Cloud prepare.
+* `subctl gather` will now use `libreswan` as  a default cable driver if none is specified in `SubmarinerSpec` during installation.
+* Sometimes when Submariner is used to connect onPrem clusters with Public clusters, MTU issues are seen. This was particularly noticed
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Sometimes when Submariner with Globalnet enabled is used to connect onPrem clusters with Public clusters, MTU issues are seen. This was particularly noticed
```

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
It is important to clarify that MTU issues are seen only with Globalnet enabled. Can you please make this change.

File: src/content/community/releases/_index.en.md (hunk 2 above)
Comment (sridhargaddam) [reply src/content/community/releases/_index.en.md]: Initially we used to use the following image `PnTAE.CPU_16_Memory_32768_Disk_80` and it was changed to `PnTAE.CPU_4_Memory_8192_Disk_50` - i.e., 4 VCPUs, 8GB RAM, 50GB disk

**Review by nyechiel — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**